sdk/core/src/main/java/com/klaviyo/coresdk/Klaviyo.kt

```diff
@@ -1,73 +1,146 @@
+     * This must be called to before using any other SDK functionality
      *
-     * Whenever the active user in an app changes (for example via a fresh login) this should be
-     * called to set the new user information in memory.
-     *
-     * @param email Email address for an active user
+     * @param apiKey - Your Klaviyo account's public API Key
+     * @param applicationContext
+     * @return
      */
-    fun setUserEmail(email: String) {
-        UserInfo.email = email
+    fun initialize(apiKey: String, applicationContext: Context) {
+        Registry.config = Registry.configBuilder
+            .apiKey(apiKey)
+            .applicationContext(applicationContext)
+            .build()
     }
 
     /**
-     * Creates a track request for the Klaviyo APIs and begins processing.
-     * Track requests track the triggering of a specified [KlaviyoEvent] in Klaviyo
+     * Assigns an email address to the current internally tracked profile
+     *
+     * The SDK keeps track of current profile details to
+     * build analytics requests with profile identifiers
+     *
+     * This should be called whenever the active user in your app changes
+     * (e.g. after a fresh login)
      *
-     * @param customerProperties A map of customer property information.
-     * Defines the customer that triggered this event
-     * @param properties A map of event property information.
-     * Additional properties associated to the event that are not for identifying the customer
+     * @param email Email address for active user
+     * @return
      */
-    fun track(
-        event: KlaviyoEvent,
-        customerProperties: KlaviyoCustomerProperties,
-        properties: KlaviyoEventProperties? = null
-    ) {
-        val request = TrackRequest(KlaviyoConfig.apiKey, event, customerProperties, properties)
-        processRequest(request)
-    }
+    fun setEmail(email: String): Klaviyo = this.setProfile(Profile().setEmail(email))
+
+    /**
+     * Assigns a phone number to the current internally tracked profile
+     *
+     * The SDK keeps track of current profile details to
+     * build analytics requests with profile identifiers
+     *
+     * This should be called whenever the active user in your app changes
+     * (e.g. after a fresh login)
+     *
+     * @param phoneNumber Phone number for active user
+     */
+    fun setPhoneNumber(phoneNumber: String): Klaviyo =
+        this.setProfile(Profile().setPhoneNumber(phoneNumber))
 
     /**
-     * Creates an identify request for the Klaviyo APIs and begins processing.
+     * Assigns an external ID to the current internally tracked profile
+     *
+     * The SDK keeps track of current profile details to
+     * build analytics requests with profile identifiers
+     *
+     * This should be called whenever the active user in your app changes
+     * (e.g. after a fresh login)
+     *
+     * @param id Phone number for active user
```

Comment:
Think you still have a copy/paste issue here since its mentioning phone number as a description for the id param

Reply:
Fixed in https://github.com/klaviyo/klaviyo-android-sdk/pull/35